float32[] color
float32 blink_on_duration
float32 blink_off_duration